test/unit/publication_test.rb

```diff
@@ -415,11 +415,11 @@ class PublicationTest < ActiveSupport::TestCase
     assert pub.valid?
     assert_equal '10.5072/abc', pub.doi
 
-    pub.doi = 'http://dx.doi.org/10.5072/abc'
+    pub.doi = 'https://doi.org/10.5072/abc'
     assert pub.valid?
```

Comment:
Could this test check both the old and new resolver URLs?

Reply:
Phew... Testing in Ruby is a bit out of my waters, so I tried duplicating the input strings & asserts for the other URL variants, see below.